src/js/about.ts

```diff
@@ -1,6 +1,6 @@
 /* global document, window */
 const setUpAbout = () => {
-  const aboutElement = document.querySelector(".about-popup");
+  const aboutElement = document.querySelector(".about-popup") as HTMLElement;
```

Comment:
FYI `as HTMLElement` is telling TypeScript "trust me, this is this type". An alternative would be to check with runtime code that it's an `HTMLElement` and error if so. That would be more robust. But it's more boilerplate, and we already have tests to check that this works properly.